res/menu/dashboard.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android" >
+    <item android:id="@+id/menu_as_refresh" android:showAsAction="ifRoom" android:title="@string/LoadingData" android:icon="@drawable/action_bar_icon_refresh"></item>
```

Comment:
The convention I used to name the menu item IDs is: `menu_$menuname_$itemname`.
`as` is short for `Activity Stream`, aka SocialTabsActivity.
Here, it's better to use `dashboard` as the menu name, so this item id becomes `menu_dashboard_refresh`.
This comment applies to the share menu as well.